src/jsonrpc/tests.rs

```diff
@@ -4,7 +4,9 @@ use url::Url;
 
 use crate::jsonrpc::{JsonRpcClient, JsonRpcClientImpl, NO_PARAMS};
 
-const HTTP_ENDPOINT: &str = "https://api.node.glif.io/rpc/v0";
+/// The default endpoints for public lotus node. If the urls fail in running tests, need to
+/// check these endpoints again.
+pub const HTTP_ENDPOINT: &str = "https://api.node.glif.io/rpc/v0";
```

Comment:
I think I'd rather keep this private and explicitly add another constant in the other `tests` module, but up to you. It just feels weird not having each test module hermetic.

Reply:
yeah, make sense. I'll revert to each module have their own parameter. Resolving this.